src/components/Workspace.jsx

```diff
@@ -65,13 +67,27 @@ function mapStateToProps(state) {
     errors: state.get('errors').toJS(),
     runtimeErrors: state.get('runtimeErrors').toJS(),
     isUserTyping: state.getIn(['ui', 'editors', 'typing']),
-    editorsFlex: state.getIn(['ui', 'editors', 'verticalFlex']).toJS(),
+    editorsFlex: state.getIn(['ui', 'workspace', 'columnFlex']).toJS(),
+    rowsFlex: state.getIn(['ui', 'workspace', 'rowFlex']).toJS(),
     currentUser: state.get('user').toJS(),
     ui: state.get('ui').toJS(),
     clients: state.get('clients').toJS(),
   };
 }
 
+function getNodeWidth(node) {
+  const minWidth = window.getComputedStyle(node).minWidth;
+  return {
+    width: node.offsetWidth,
+    minWidth: parseInt(minWidth.replace('px', ''), 10),
+  };
+}
+
+function getNodeWidths(refs) {
```

Comment:
I would still lean toward making these named exports of the `resizer` module, even if we won’t be reusing it, just to reduce the amount of drag-to-resize support code we need in the components. Also has the advantage of fully abstracting the question of “what does the resizer need to know about node dimensions?” from the components.